src/components/TodosContext/TodosContext.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { ChangeEvent, useState } from 'react';
+
+type TodosContextProps = {
```

Comment:
```suggestion
interface TodosContextProps = {
```
Use interface prefer.